napari/components/dims.py

```diff
@@ -171,6 +171,10 @@ def point(self) -> Tuple[int, ...]:
         )
         return point
 
+    @point.setter
+    def point(self, point: Tuple[int, ...]) -> None:
+        self.point = point
```

Comment:
This will cause an infinite recursion when used.

Before this PR, `point` is a derived property that depends on `range` and `current_step`. If you want a setter for it, I think it should effectively modify `current_step`.